Example/OriginateUI/CircleViewController.m

```diff
@@ -0,0 +1,89 @@
+//
+//  CircleViewController.m
+//  OriginateUI
+//
+//  Created by Seth Rainha on 7/7/16.
+//  Copyright © 2016 originate.com. All rights reserved.
+//
+
+#import "CircleViewController.h"
+#import "CodeViewController.h"
+@import OriginateUI;
+
+@interface CircleViewController ()
+
+@property (nonatomic, strong) UILabel *mainLabel;
+@property (nonatomic, strong) UILabel *secondLabel;
+
+@end
+
+@implementation CircleViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.view.backgroundColor = [UIColor whiteColor];
+    CGFloat screenHeight = CGRectGetHeight(self.view.frame);
+    CGFloat screenWidth = CGRectGetWidth(self.view.frame);
+    self.title = @"Circle Images";
+    
+    UIImageView *imageView =[[UIImageView alloc] initWithFrame:CGRectMake(screenWidth*0.5-50,screenHeight*0.35, 100, 100)];
+    CGSize mySize = CGSizeMake(50, 50);
+    UIImage *firstImage = [UIImage circularImageWithText:@"S.R" textColor:[UIColor blueColor] backgroundColor:[UIColor hex:0xFF2D55] size:mySize];
+    imageView.image = firstImage;
+    [self.view addSubview:imageView];
```

Comment:
Programmatically set up view hierarchy is supposed to be implemented in `loadView`.